sources/Algo/src/Algo.cpp

```diff
@@ -17,6 +17,7 @@
 
 #include "Algo.h"
 
+#include "DYNCommon.h"
```

Comment:
formatting not done properly. Also, since it is an exterior header, you should use <>

Reply:
What do you meant about formatting ?

Reply:
formatting by clang-format